src/group_theory/nielsen_schreier.lean

```diff
@@ -174,7 +174,7 @@ end
 /-- Since a hom gives a loop, any homomorphism from the vertex group at the root
     extends to a functor on the whole groupoid. -/
 @[simps] def functor_of_monoid_hom {X} [monoid X] (f : End (root' T) →* X) :
-  G ⥤ single_obj X :=
+  G ⥤ category_theory.single_obj X :=
```

Comment:
What's the point of introducing the `category_theory.single_obj` abbreviation if you need to fully qualify the name afterwards because else it would pick up `quiver.single_obj`?

Reply:
In most category theory files where `category_theory.single_obj` is used, the quiver namespace is not open (nielsen_scheier is the only exception). The abbreviation avoids having to modify all these files.